docs/ObjectStoreAccessKeysApi.md

```diff
@@ -64,7 +73,7 @@ Name | Type | Description  | Notes
 [[Back to top]](#) [[Back to API list]](index.md#endpoint-properties) [[Back to Model list]](index.md#documentation-for-models) [[Back to Overview]](index.md)
 
 # **delete_object_store_access_keys**
-> delete_object_store_access_keys(ids=ids, names=names)
+> delete_object_store_access_keys(names=names)
```

Comment:
Had questions about this (`ids` being dropped for some operations in this file) and had @bsamz-ps confirm that this was correct.